dom/dom_c.php

```diff
@@ -128,13 +128,13 @@ class DOMNode  {
      * @param DOMNode $node <p>
      * The new node.
      * </p>
-     * @param DOMNode $child [optional] <p>
+     * @param null|DOMNode $child [optional] <p>
      * The reference node. If not supplied, newnode is
      * appended to the children.
      * </p>
      * @return DOMNode The inserted node.
      */
-    public function insertBefore (DOMNode $node, DOMNode $child = null) {}
+    public function insertBefore (DOMNode $node, ?DOMNode $child = null) {}
```

Comment:
There is a tricky case here. Class DOMNode is available since PHP 5 but nullable types are available only since PHP 7.1 so if one will try to overwrite this method in PHP < 7.1 then one will get a warning `Declaration should be compatible with DOMNode->insertBefore` and thus such changes should be avoided. Nullable type hints can be added only in final classes and methods that can't be overwritten.